src/foam/mlang/mlang.js

```diff
@@ -3775,6 +3776,137 @@ foam.CLASS({
   ]
 });
 
+foam.CLASS({
```

Comment:
@kgrgreer @KernelDeimos Any requirement that mlang classes be in the same source/model file?